pilot/pkg/serviceregistry/providers.go

```diff
@@ -14,16 +14,16 @@
 
 package serviceregistry
 
-// ServiceRegistry defines underlying platform supporting service registry
-type ServiceRegistry string
+// ProviderID defines underlying platform supporting service registry
+type ProviderID string
```

Comment:
why not simply `Provider` ?

Reply:
I considered it, but `ID` makes it more obvious that it's just a string, rather than an implementation of a service registry